tchannel-core/src/test/java/com/uber/tchannel/tracing/TracingPropagationTest.java

```diff
@@ -323,9 +320,9 @@ private static TraceResponse callDownstreamThrift(String remainingEncodings, Str
 
     @Test
     public void testPropagation() throws Exception {
-        Span span = (Span) tracer.buildSpan("root").startManual();
-        SpanContext context = span.context();
-        String traceId = String.format("%x", context.getTraceId());
+        Span span = tracer.buildSpan("root").start();
+        JaegerSpanContext context = (JaegerSpanContext) span.context();
+        String traceId = context.getTraceId();
```

Comment:
I suggest using `toTraceId()` consistently everywhere, not mixing get- and to-

Reply:
For this line, we can use SpanContext.toTraceId(), but there are a few cases where JaegerSpanContext is needed e.g. the code needs to know if the span is sampled or not (Line:231).

Do you want to have this?
`
       	 response.traceId = span.context().getTraceId();
         JaegerSpanContext context = (JaegerSpanContext) span.context();
          response.sampled = context.isSampled();`

Reply:
Make sense. I thought SpanContext provides toTraceId while JaegerSpanContext provides getTraceId. Just found out both have toTraceId function.

Updated.